### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by deliahu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,8 +115,8 @@ positive
 ```bash
 $ cortex get sentiment-classifier --watch
 
-status   up-to-date   requested   last update   avg inference   2XX
```
Comment (vishalbollu) [inline README.md]: Good catch

File: docs/deployments/api-configuration.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+# API configuration
+
+_WARNING: you are on the master branch, please refer to the docs on the branch that matches your `cortex version`_
+
+Once your model is [exported](exporting.md) and you've implemented a [Predictor](predictors.md), you can configure your API via a yaml file (typically named `cortex.yaml`).
+
+Reference the section below which corresponds to your Predictor type: [Python](#python-predictor), [TensorFlow](#tensorflow-predictor), or [ONNX](#onnx-predictor).
```
Comment (vishalbollu) [inline docs/deployments/api-configuration.md]: Pretty cool that these links take you to the correct section of the page

File: docs/deployments/predictors.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+tensorflow-hub==0.7.0
+tensorflow==2.1.0
+```
+
+<!-- CORTEX_VERSION_MINOR -->
+The pre-installed system packages are listed in [images/tf-api/Dockerfile](https://github.com/cortexlabs/cortex/tree/master/images/tf-api/Dockerfile).
+
+If your application requires additional dependencies, you can install additional [Python packages](python-packages.md) and [system packages](system-packages.md).
+
+### Examples
```
Comment (vishalbollu) [inline docs/deployments/predictors.md]: I would place the examples with a sample implementation of the predictor before show pre-installed packages.

File: docs/deployments/python-packages.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Python packages
```
Comment (vishalbollu) [inline docs/deployments/python-packages.md]: Include setup.py for completeness?

Comment (deliahu) [reply docs/deployments/python-packages.md]: Good idea, I'll do that in a separate PR

**Pull request merged**